server/views/pages/log-application/application-details/swap-visiting-orders-for-pin-credit.njk

```diff
@@ -14,12 +15,24 @@
       href: "/log/prisoner-details"
     }) }}
 
-    <span class="govuk-caption-xl">{{ appTypeTitle }}</span>
-    <h1 class="govuk-heading-xl govuk-!-margin-top-0">Log details</h1>
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-one-half">
+          {% if errors %}
```

Comment:
Which IDE do you use @sweetymj I think the tab setting is set to 4 characters instead of 2?